content/docs/authorization/deployment/_index.md

```diff
@@ -51,38 +55,51 @@ A Storage Administrator can execute the installer or rpm package as a root user
         "host": ":8080"
       },
       "zipkin": {
-        "collectoruri": "http://DNS_host_name:9411/api/v2/spans",
+        "collectoruri": "http://DNS-hostname:9411/api/v2/spans",
         "probability": 1
       },
       "certificate": {
         "keyFile": "path_to_private_key_file",
         "crtFile": "path_to_host_cert_file",
         "rootCertificate": "path_to_root_CA_file"
       },
-      "hostName": "DNS_host_name"
+      "hostname": "DNS-hostname"
     }
     ```
 
-    In the above template, `DNS_host_name` refers to the hostname of the system in which the CSM for Authorization server will be installed. This hostname can be found by running the below command on the system:
+    In an instance where a secure deployment is not required, an insecure deployment is possible. Please note that self-signed certificates will be created for you using cert-manager to allow TLS encryption for communication on the CSM for Authorization proxy server. However, this is not recommended for production environments. For an insecure deployment, the json file in the location `$HOME/.karavi/config.json` only requires the following contents:
 
-    ```
-    nslookup <IP_address>
+    ```json
+    {
+      "hostname": "DNS-hostname"
+    }
     ```
 
-2. In order to configure secure grpc connectivity, an additional subdomain in the format `grpc.DNS_host_name` is also required. All traffic from `grpc.DNS_host_name` needs to be routed to `DNS_host_name` address, this can be configured by adding a new DNS entry for `grpc.DNS_host_name` or providing a temporary path in the `/etc/hosts` file.  
+>__Note__:
+> - `DNS-hostname` refers to the hostname of the system in which the CSM for Authorization server will be installed. This hostname can be found by running `nslookup <IP_address>`
+> - There are a number of ways to create certificates. In a production environment, certificates are usually created and managed by an IT administrator. Otherwise, certificates can be created using OpenSSL.
 
->__Note__: The certificate provided in `crtFile` should be valid for both the `DNS_host_name` and the `grpc.DNS_host_name` address.  
+2. In order to configure secure grpc connectivity, an additional subdomain in the format `grpc.DNS-hostname` is also required. All traffic from `grpc.DNS-hostname` needs to be routed to `DNS-hostname` address, this can be configured by adding a new DNS entry for `grpc.DNS-hostname` or providing a temporary path in the systems `/etc/hosts` file. 
 
-    For example, create the certificate config file with alternate names (to include example.com and grpc.example.com) and then create the .crt file:  
+    ```json
```

Comment:
Is this snippet representing the /etc/hosts file?

Reply:
No, this is an example for adding the grpc entry in `.karavi/config.json`

Reply:
I misunderstood the DNS server configuration, removed